source/globus/globus_platform.rst

```diff
@@ -0,0 +1,70 @@
+.. raw:: html
+
+   <style> .bgrnd1 {background-color:#DCDCDC; border-radius:5px} </style>
+
+.. role:: bgrnd1
+
+
+.. _globus_platform:
+
+Globus file and data sharing platform
+=====================================
+
+The Globus platform enables developers to provide robust file transfer, sharing and search capabilities within their own research data applications and services, while leveraging advanced identity management, single sign-on, and authorization capabilities.
+
+This document is a hands-on guide to the Globus file sharing platform. It should complement the official documentation at `docs.globus.org <https://docs.globus.org>`_.
+
+For all questions concerning the Globus file sharing platform, please contact the VSC Globus team at data@vscentrum.be.
+
+.. include:: globus_contact_information.rst
+
+We welcome your feedback, comments and suggestions for improving the Globus Tutorial.
+
+
+.. _access:
+
+Access
+------
+
+.. include:: access.rst
+
+
+.. _managing_and_transferring_files:
+
+Managing and transferring files
+-------------------------------
+
+.. include:: managing_and_transferring_files.rst
+
+
+.. local_endpoints
+
+Local endpoints
+---------------
+
+.. include:: local_endpoints.rst
+
+
+.. _data_sharing
+
```

Comment:
Add a use case for Tier-2/Tier-1 data transfer?

Reply:
I am not sure I follow. We have this - Note At the VSC all Tier-1 and Tier-2 systems have dedicated Globus Collections. Type vsc in the search field to see the list.